code/modules/research/designs/spacepod_designs.dm

```diff
@@ -0,0 +1,318 @@
+//////SPACEPOD GUNS///////////////////////
+//////////////////////////////////////////
+
+/datum/design/pod_gun_disabler
+	construction_time = 200
+	name = "Spacepod Equipment (Disabler)"
+	desc = "Allows for the construction of a spacepod mounted disabler."
+	id = "podgun_disabler"
+	build_type = PODFAB
+	req_tech = list("materials" = 2, "combat" = 2)
+	build_path = /obj/item/device/spacepod_equipment/weaponry/disabler
+	category = list("Pod Weaponry")
+	materials = list(MAT_METAL = 15000)
+
+/datum/design/pod_gun_disabler
+	construction_time = 200
+	name = "Spacepod Equipment (Burst Disabler)"
+	desc = "Allows for the construction of a spacepod mounted disabler. This is the burst-fire model."
+	id = "podgun_bdisabler"
+	build_type = PODFAB
+	req_tech = list("materials" = 3, "combat" = 3)
+	build_path = /obj/item/device/spacepod_equipment/weaponry/burst_disabler
+	category = list("Pod Weaponry")
+	materials = list(MAT_METAL = 15000,MAT_PLASMA=2000)
+
+/datum/design/pod_gun_laser
+	construction_time = 200
+	name = "Spacepod Equipment (Laser)"
+	desc = "Allows for the construction of a spacepod mounted laser."
+	id = "podgun_laser"
+	build_type = PODFAB
+	req_tech = list("materials" = 3, "combat" = 3, "plasmatech" = 2)
+	build_path = /obj/item/device/spacepod_equipment/weaponry/laser
+	category = list("Pod Weaponry")
+	materials = list(MAT_METAL=10000,MAT_GLASS=5000,MAT_GOLD=1000,MAT_SILVER=2000)
+
+/datum/design/pod_mining_laser_basic
+	construction_time = 200
+	name = "Basic Mining Laser"
+	desc = "Allows for the construction of a weak mining laser"
+	id = "pod_mining_laser_basic"
+	req_tech = list("materials" = 3, "powerstorage" = 2, "engineering" = 2, "magnets" = 3, "combat" = 2)
+	build_type = PODFAB
+	materials = list(MAT_METAL = 10000, MAT_GLASS = 5000, MAT_SILVER = 2000, MAT_URANIUM = 2000)
+	build_path = /obj/item/device/spacepod_equipment/weaponry/mining_laser_basic
+	category = list("Pod Weaponry")
+
+/datum/design/pod_mining_laser_basic
+	construction_time = 200
+	name = "Basic Kinetic Accelerator"
+	desc = "Allows for the construction of a weak spacepod Kinetic Accelerator"
+	id = "pod_mining_laser_basic"
+	req_tech = list("materials" = 3, "powerstorage" = 2, "engineering" = 2, "magnets" = 3, "combat" = 2)
+	build_type = PODFAB
+	materials = list(MAT_METAL = 10000, MAT_GLASS = 5000, MAT_SILVER = 2000, MAT_URANIUM = 2000)
+	build_path = /obj/item/device/spacepod_equipment/weaponry/mining_laser_basic
+	category = list("Pod Weaponry")
+
+/datum/design/pod_mining_laser
+	construction_time = 200
+	name = "Kinetic Accelerator"
+	desc = "Allows for the construction of a spacepod Kinetic Accelerator."
+	id = "pod_mining_laser"
+	req_tech = list("materials" = 6, "powerstorage" = 6, "engineering" = 5, "magnets" = 6, "combat" = 4)
+	build_type = PODFAB
+	materials = list(MAT_METAL = 10000, MAT_GLASS = 5000, MAT_SILVER = 2000, MAT_GOLD = 2000, MAT_DIAMOND = 2000)
+	build_path = /obj/item/device/spacepod_equipment/weaponry/mining_laser
```

Comment:
Be sure to update the items in each of these designs to be kinetic accelerators and not mining lasers.